README.md

```diff
@@ -240,6 +241,10 @@ This is the backend API repository for TurLink. TurLink is a link shortener app
       ]
     }
     ```
+  ### Click Tracking
+  - Each time a shortened link is accessed via the GET `/api/v1/links?short={shortened link}` endpoint:
+    - The `click_count` for that link is automatically incremented.
+    - The `last_click` timestamp is updated to the current time.
```

Comment:
Might want to move this into the "Return full link when short link is given" section since it's not its own endpoint. Maybe a subsection within that section?